src/MCPClient/lib/clientScripts/copySubmissionDocs.sh

```diff
@@ -8,3 +8,4 @@
 subdir="$1/submissionDocumentation"
 mkdir -p $subdir
 cp -R "$1/$2/data/objects/submissionDocumentation" $subdir
```

Comment:
Yep, but if we adopt `set -e` then the script fails if any individual line exits non-1 - failures get treated like unhandled exceptions in Python. We don't have to do that right away, but it seems like something I'd like to do in the future, and getting scripts ready for that where we touch them would make it easier to handle that in the future.